assets/src/edit-story/components/library/panes/media/media3p/categoryPill.js

```diff
@@ -51,19 +56,28 @@ PillContainer.propTypes = {
   isSelected: PropTypes.bool,
 };
 
-const CategoryPill = (props) => (
-  <PillContainer
-    isSelected={props.isSelected}
-    onClick={props.onClick}
-    role="tab"
-    aria-selected={props.isSelected}
-    data-testid="mediaCategory"
-  >
-    {props.title}
-  </PillContainer>
-);
+const CategoryPill = (props) => {
```

Comment:
nit: I think it would be, due to the amount of args, easier to read if they'd be deconstructed here.

Reply:
+1, we add 'props' implicitly to too many components. Updated.